zest/releaser/tests/bzr.txt

```diff
@@ -58,9 +58,11 @@ Commit it:
     >>> cmd
     'bzr commit -v -m "small tweak"'
     >>> print(system(cmd))
-    Committing to: TESTTEMP/tha.example-bzr/
-    modified setup.py
-    Committed revision 2.
+    RED Committing to: TESTTEMP/tha.example-bzr/
```

Comment:
Yes, it would be. But this is simply the `bzr` command that prints this to `stderr` where I think `stdout` would have been more appropriate... There are more commands that have chosen differently, so we do gain a few false positives here.

A few lines standing out may not be so bad. Alternatively, we would need to refactor our `system` function to actually read return codes to determine if something went wrong, instead of checking `stderr`.